### PR TITLE
[#8605] Style plan detail page 

### PR DESCRIPTION
**Pull request by sevfurneaux**
**Describe your changes**
This PR styles the plan detail page and adds the React `<ProjectsList />` component for projects.

Screenshot 2024-12-12 at 22 47 44

Screenshot 2024-12-12 at 22 47 54

**Tasks**
- [X] PR name contains story or task reference
- [ ] Steps to recreate and test the changes
- [ ] Documentation (docs and inline)
- [ ] Tests (including n+1 and django_assert_num_queries where applicable)
- [X] Changelog

### CODE REVIEW

**Comment by sevfurneaux** [comment]
@vellip @hom3mad3 Looks like my [2 line Python code](https://github.com/liqd/a4-meinberlin/pull/5888/files#diff-5d1d34d0d6492d67ca9b08e768984e5b029cbb52f321c32b3c0331f36379e200R55-R58) decreased the test coverage by `-0.006%` 🤣

Do I need to write a Python test? 😅

File: meinberlin/apps/plans/templates/meinberlin_plans/plan_detail.html [hunk 1]
```diff
@@ -42,29 +42,58 @@
 
 {% block content %}
 <div id="layout-grid__area--maincontent">
+    {% if object.image %}
+        <section class="herounit-image-with-aside fullwidth">
+            <div class="mainbar">
+                <div class="mainbar__left">
+                    <div class="image">
+                        <div class="image__image">
+                            <img src="{{ object.image | thumbnail_url:"heroimage" }}" alt="{% if object.image_alt_text %}{{ object.image_alt_text }}{% else %}{% translate "Here you can find a decorative picture." %}{% endif %}" width="100%" height="auto" loading="lazy"/>
```
[the remark this answers is not in the document]
Comment (vellip) [reply meinberlin/apps/plans/templates/meinberlin_plans/plan_detail.html:51]: Do you think having this non-descriptive text is better than an empty alt tag? If I imagine using this page with a screen reader I think I would consider this clutter.

File: meinberlin/apps/plans/views.py [hunk 2]
```diff
@@ -50,6 +52,10 @@ class PlanDetailView(rules_mixins.PermissionRequiredMixin, CanonicalURLDetailVie
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context["polygon"] = settings.BERLIN_POLYGON
+        serializer = project_serializers.ProjectSerializer(
+            self.object.published_projects, many=True, now=timezone.now()
+        )
+        context["published_projects"] = json.dumps(serializer.data)
```
Comment (m4ra) [inline meinberlin/apps/plans/views.py:58]: @sevfurneaux did you try to add the `published_projects` to the [Plan serializer](https://github.com/liqd/a4-meinberlin/blob/main/meinberlin/apps/plans/serializers.py#L10) instead? I'd advice to do so, as this is rather hacky here. 
Then please a line to assert the list of the published projects [here](https://github.com/liqd/a4-meinberlin/blob/main/tests/plans/test_serializer.py#L88).
Let me know if it's too much and I will take over this in another PR.

**Review by m4ra — CHANGES_REQUESTED** [review]
a new serializer field `published_projects` should go in the Plan serialiser. See my inline comment.

File: meinberlin/assets/scss/components_user_facing/_item_detail.scss [hunk 3]
```diff
@@ -1,8 +1,35 @@
...
+}
+
+.item-detail__edit-link {
+    width: auto;
+    font-size: $em-spacer;
+    line-height: 1.2;
+    display: inline-block;
+    text-align: center;
+    cursor: pointer;
+    overflow: visible;
```
[the remark this answers is not in the document]
Comment (sevfurneaux) [reply meinberlin/assets/scss/components_user_facing/_item_detail.scss]: Thanks! Removed 👍

File: meinberlin/apps/plans/views.py (hunk 2 above)
Comment (m4ra) [reply meinberlin/apps/plans/views.py:58]: I will see if I have time later or tomorrow morning. Otherwise I will merge, and make a new issue for later.

Comment (sevfurneaux) [reply meinberlin/apps/plans/views.py:58]: Excellent, thanks!

**Pull request merged**